web/src/pages/speedtest.rs

```diff
@@ -336,33 +340,35 @@ impl Speedtest {
     ) -> Html {
         let tps = (confirmed_tx as u64 * 1000).checked_div(elapsed.num_milliseconds() as u64);
 
-        html! { <MatList>
-            <MatListItem> { Speedtest::progress_bar(
-                "Transactions sent",
-                (sent_tx as f32)/(total_tx as f32)
-            ) } </MatListItem>
-            <MatListItem> { Speedtest::progress_bar(
-                "Transactions confirmed_tx",
-                (confirmed_tx as f32)/(total_tx as f32)
-            ) } </MatListItem>
-
-            <MatListItem>
-                <p> { format!("Running for {}s", elapsed.num_seconds()) } </p>
-            </MatListItem>
-
-            <MatListItem>
-                <p> { "AT2's computed TPS: " } {
-                    tps
-                        .map(|tps| html! { tps })
-                        .unwrap_or(html! { <span style="color: lightgrey"> { "computing" } </span> })
-                } </p>
-            </MatListItem>
-            <MatListItem>
-                <p> { "Bitcoin's TPS: 7" } </p>
-            </MatListItem>
-            <MatListItem>
-                <p> { "Ethereum's TPS: 25" } </p>
-            </MatListItem>
-        </MatList> }
+        const FIRST_COL: &str = "text-align: end; padding: 0 1em";
+
+        html! { <table>
```

Comment:
Does that have `border=0`? I wouldn't want to see the grid of the table.

Reply:
It doesn't have it explicitly, but `border: 0px` is the default.